README.md

```diff
@@ -31,38 +34,57 @@ semantic data transformation &amp; semantic container crawling
 - R Component
 - Workflow sharing by Google Drive
 
-## install
+
+## Install
 
 - `ssh-keygen -t rsa`
-- ajouter la clef publique généré (.ssh/id_rsa.pub) dans l'admin de github
+- Add the generated public key (`.ssh/id_rsa.pub`) in Github admin
+
+See https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
+
+
+### Git
+
+#### Option 1
 
-### GIT
-#### option 1
+```bash
```

Comment:
I used this syntax because it is easier to copy/paste several lines without issue.

And I think it is easier to see the block.